packages/ui-patterns/CommandMenu/api/CommandList.tsx

```diff
@@ -29,7 +29,7 @@ const CommandList = forwardRef<
   return (
     <CommandList_Shadcn_
       ref={setRef}
-      className={cn('overflow-y-auto overflow-x-hidden bg-transparent', className)}
+      className={cn('max-h-[initial] overflow-y-auto overflow-x-hidden bg-transparent', className)}
```

Comment:
Not sure if this is intentional and not sure if this is where it happens, but the width of the modal changes between sections.

https://github.com/user-attachments/assets/b4a3a63b-db34-4494-9ae2-30d8834b85dd

Reply:
And it looks like some additional padding snuck in the ai chat

![Screenshot 2024-07-18 at 10 29 53](https://github.com/user-attachments/assets/c8ff9baf-d210-4f36-a128-8a899fbc3ac0)

Reply:
The resizing one is intentional (people wanted more room to read the AI responses), but let me take a look at the padding...